src/utils/iconify.ts

```diff
@@ -172,3 +172,13 @@ addIcon("custom:md:dog-white", {
   width: 30,
   height: 30,
 });
+addIcon("custom:md:twitter-logo", {
```

Comment:
Bro this this `x` not `twitter` anymore lmao

Reply:
gg